config/locales/en.yml

```diff
@@ -3062,7 +3062,7 @@ en:
                   data-track-attribute-position="stimulus-how-faq3">
                 Get My Payment</a> tool will show a status of “more information needed” and you will have the opportunity to provide new bank information.
             - title: What if I haven’t filed federal taxes?
-              content_html: We recommend that you file your taxes in order to access the Economic Impact Payments and any additional tax credits you may be eligible for. You can file 2017, 2018, 2019, and 2020 returns for free with GetYourRefund.
+              content_html: We recommend that you file your taxes in order to access the Economic Impact Payments and any additional tax credits you may be eligible for. You can file 2018, 2019, 2020, and 2021 returns for free with GetYourRefund.
```

Comment:
i was sad to learn that you can't use interpolation for these list-y things

Reply:
You can -- you can just pass the variable in at the top level:

```
<% t("whatever.content_html", years: years) do |content| %>
  <p><%= content %></p>
<% end %>  


```

Reply:
oh hm i tried that and it just displayed %{years} on the page. do we have an example of this somewhere in the code? i couldn't find anything on the internet (with a cursory search anyway)

Reply:
there aren't many upvotes here but if this is to be believed, it looks like it's not supported: https://stackoverflow.com/questions/21574900/interpolation-in-i18n-array

Reply:
Ah -- the issue is that you can send things into arrays, but the implementation on that page has arrays with then a nested array of attributes, and we're accessing the data on that nested array directly instead of using the translation string. 

```
<% t("whatever.content_html", years: years).each do |content| %> <p><%= content[:item] %></p> <% end %>
```
That content[:item] part cant access the variables